docs/src/DeveloperDocumentation/changelog.md

```diff
@@ -0,0 +1,114 @@
+| topic: groups                 | Changes related to Groups |
+| topic: LieTheory              | Changes related to Lie Theory |
+| topic: number theory          | Changes related to Number Theory |
+| topic: polyhedral geometry    | Changes related to Polyhedral Geometry |
+| topic: schemes                | Changes related to Schemes |
+| topic: toric schemes          | Changes related to Toric Schemes |
+| topic: toric varieties        | Changes related to Toric Varieties |
+| topic: tropical geometry      | Changes related to Tropical Geometry |
+| serialization                 | Changes related to serializing data in the MRDI file format ? |
+| enhancement                   | New features or extended functionality |
+| experimental                  | Only changes experimental parts of OSCAR |
+| optimization                  | Performance improvements or improved testing |
+| bug: crash                    | Fixed bugs that could lead to crashes |
+| bug                           | Other fixed bugs |
+| documentation                 | Improvements or additions to documentation |
+| package: AbstractAlgebra      | Changes related to the package AbstractAlgebra |
+| package: AlgebraicSolving     | Changes related to the package AlgebraicSolving |
+| package: GAP                  | Changes related to the package GAP |
+| package: Hecke                | Changes related to the package Hecke |
+| package: Nemo                 | Changes related to the package Nemo |
+| package: Polymake             | Changes related to the package Polymake |
+| package: Singular             | Changes related to the package Singular |
+
+
+## How to set labels when submitting or reviewing a PR
+
+Everyone submitting a PR should ideally follow this checklist if they also have the access
+rights for assigning labels. If they don't, they should ideally leave instructions that
+indicate which labels to set, and a PR reviewer should then perform the following steps.
+
+1. If your PR should *not* appear in the changelog, assign the `release notes: not needed` label and stop here.
+2. If possible, edit the title of your PR to be usable as an entry in the changelog, and assign the `release notes: use title` label.
+   - Hint: Try to start the title with a verb in present tense such as "Add", "Enhance", "Remove".
+   - Hint: if function or type names appear in the title, surround them by backticks.
+3. If you need a longer description than fits into the title, leave a comment with your suggestion
+   for this description, and assign the `release notes: to be added` label.
+4. In either case, also assign at least one `topic: XYZ` label, and possible some of the other
+   labels describe in the previous section as you deem fit.
+
+Optionally, if you think your change is exceptionally noteworthy, also assign the
+`release notes: highlight` label.
+
+
+## Example
+
+For example, if the is a PR with title `Bump GAP.jl to 0.13`, and labels `package: GAP` and
```

Comment:
Something seems to be missing between "the" and "is".

Reply:
Perhaps it should read:
```suggestion
For example, if the change is a PR with title `Bump GAP.jl to 0.13`, and labels `package: GAP` and
```